pylab/website/urls.py

```diff
@@ -11,6 +11,8 @@
     url(r'^projects/create/$', views.project_create, name='create-project'),
     url(r'^projects/(?P<project_slug>%s)/$' % slug, views.project_details, name='project-details'),
     url(r'^projects/(?P<project_slug>%s)/update/$' % slug, views.project_update, name='project-update'),
+    url(r'^events/(?P<year>[0-9]{4})/(?P<month>[0-9]{2})/(?P<day>[0-9]+)/(?P<event_slug>%s)/$' % slug,
+        views.event_details, name='event-details'),
```

Comment:
If we changed week day of the meetings, we could only change this hardcoded line ```next_monday = next_weekday(0)```, instead of renaming all the variables. I think, ```weekly``` instead of ```monday``` is better.